srcpkgs/libosinfo/template

```diff
@@ -11,15 +11,15 @@ configure_args="-Denable-introspection=$(vopt_if gir enabled disabled)
 hostmakedepends="glib-devel intltool pkg-config wget hwids
  $(vopt_if gtk_doc gtk-doc)
  $(vopt_if vala vala)"
-makedepends="libxslt-devel libglib-devel libsoup-devel"
+makedepends="libxslt-devel libglib-devel libsoup3-devel"
```

Comment:
We need to be careful about switching to libsoup3, let's not do it now.

Reply:
Alright, I just saw in the changelog that it's preferred over libsoup2. Is libsoup3 still unstable?